src/collections/members/daniel-kiptoon/index.mdx

```diff
@@ -0,0 +1,20 @@
+---
+name: Daniel Kiptoon
+position: Contributor
+image_path: ./daniel-kiptoon.jpg
+github: kiptoonkipkurui
+twitter: danchi101
+linkedin: daniel-kipkurui-kiptoon-56b8a1177
+layer5: a1837b00-c5b8-4a91-86d6-dec9527d70be
+location: Nairobi, Kenya
+bio: "Daniel Kiptoon is a seasoned software engineer with a fervent passion for creating scalable systems, mastering the intricacies of distributed computing (Kubernetes and Docker), and promoting DevSecOps principles. He aspires to improve the technology landscape with innovative solutions and to lead the conversations on the next generation cloud-native solutions."
+badges: 
+   - meshery-operator
+blurb: Distributed and Parallel
+status: Active
+maintainer: no
```

Comment:
```suggestion
```

Reply:
> @vishalvivekm I wonder though why its made me a MeshMate even though I haven't explicitly configured it in the template

@KiptoonKipkurui removing this line fixes it. Seems to me a potential bug though.

Reply:
> @vishalvivekm I wonder though why its made me a MeshMate even though I haven't explicitly configured it in the template

@KiptoonKipkurui removing this line fixes it. Seems to me a potential bug though.

Reply:
My bad. I meant `meshmate: no` line @KiptoonKipkurui